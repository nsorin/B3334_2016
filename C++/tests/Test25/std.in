LOAD t2
UNDO
LOAD t1
UNDO
REDO
HIT rect1 20 20
HIT rect1 200 200
HIT reu1 20 20
HIT reu1 200 200
HIT r1 7 7
HIT r1 3 3
EXIT
