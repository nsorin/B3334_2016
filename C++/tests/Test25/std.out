#LOAD t2
OK
#UNDO
OK
#LOAD t1
OK
#UNDO
OK
#REDO
OK
#HIT rect1 20 20
YES
#HIT rect1 200 200
NO
#HIT reu1 20 20
YES
#HIT reu1 200 200
NO
#HIT r1 7 7
YES
#HIT r1 3 3
NO
#EXIT
